iina/PlayerCore.swift

```diff
@@ -582,17 +598,22 @@ class PlayerCore: NSObject {
     info.disableOSDForFileLoading = true
     guard let path = mpvController.getString(MPVProperty.path) else { return }
     info.currentURL = URL(fileURLWithPath: path)
+    backgroundQueueTicket += 1
```

Comment:
Why? It's designed to interrupt a running block. If we post this to the same serialized queue, it can't affect any running tasks. Accesses to this property should be synchronized though.